apps/web/src/pages/templates/shared/errors.ts

```diff
@@ -69,6 +69,30 @@ export function getStepErrors(index: number | string, errors?: FieldErrors<IForm
   return [];
 }
 
+export function getVariantErrors(
+  stepIndex: number | string,
+  errors?: FieldErrors<IForm>
+): { errorMsg: string; variantIndex: number }[] {
+  if (errors?.steps) {
+    const variantsErrors = errors.steps[stepIndex]?.variants?.reduce((acc, variant, variantIndex) => {
+      const variantErrors = variant?.template;
+      if (variantErrors) {
+        const keys = Object.keys(variantErrors);
+
+        const errorMsg = keys.map((key) => variantErrors[key]?.message);
+
+        acc.push(...errorMsg.map((msg) => ({ errorMsg: msg, variantIndex })));
```

Comment:
single variant can have multiple errors
for example:
1st error: message title is missing
2nd error: message content is missing
we want to have all these errors separately